packages/frontend-core/src/utils/json.js

```diff
@@ -119,3 +119,36 @@ const extractJSONSchemaKeys = (jsonSchema, squashObjects = false) => {
   })
   return keys
 }
+
+const isObject = test => {
+  return (
+    typeof test === "object" &&
+    !Array.isArray(test) &&
+    test !== null &&
+    !(test instanceof Date)
+  )
+}
```

Comment:
This feels incomplete. For example, `isObject(new Map())` would return true.

Reply:
I took a look at some other implementations of `isObject` on StackOverflow and NPM and there doesn't seem to be any canonical implementation that people agree on (the `is-object` npm package has the same `new Map()` problem).

What's the goal of the function? Maybe it's fine that it's incomplete?

Reply:
If passing an empty object returns false, it seems like the function name is wrong. Could it be more verbose? What are you actually trying to get from it?

Reply:
I think it probably is, specifically it's testing a a property of an API JSON response body is an object, i.e. in the form:

```json
"objProp": {
  "name": "Alice",
  "age": 30
}
```

I don't think we ever cast to an instance of Map.

Reply:
Renamed to `hasSchema` to better indicate the purpose of the function.